src/libs/ReportUtils.js

```diff
@@ -883,7 +883,8 @@ function buildOptimisticIOUReport(ownerEmail, userEmail, total, chatReportID, cu
  */
 function getIOUReportActionMessage(type, total, participants, comment, currency, paymentType = '', isSettlingUp = false) {
     const amount = NumberFormatUtils.format(preferredLocale, total / 100, {style: 'currency', currency});
-    const displayNames = _.map(participants, participant => getDisplayNameForParticipant(participant.login, true));
+    const parser = new ExpensiMark();
+    const displayNames = _.map(participants, participant => parser.replace(getDisplayNameForParticipant(participant.login, true)));
```

Comment:
Similar to the above change, We should parse comment (optional message) as well.

Reply:
@tienifr

Reply:
@sobitneupane I have pushed a new commit. Please help me check it. Thank you!